packages/graph/hash_graph/lib/graph/src/store/query/filter.rs

```diff
@@ -128,6 +128,21 @@ impl<'q> Filter<'q, EntityProperties> {
         ])
     }
 
+    /// Creates a `Filter` to search for a specific entities at their latest version, identified by
+    /// its [`EntityUuid`].
+    #[must_use]
+    pub fn for_latest_entity_by_entity_uuid(entity_uuid: EntityUuid) -> Self {
```

Comment:
Is this being used anywhere? (i could find it used in `representative_read`, should we put it behind cfg?)

Reply:
(Discussed offline, this is not using ownedbyid)